src/components/numeral-date/numeral-date.component.tsx

```diff
@@ -477,67 +473,149 @@ export const NumeralDate = <DateType extends NumeralDateObject = FullDate>({
-                      </FormSpacingProvider>
+                      <StyledDateLabel
+                        id={labelId}
+                        data-role="numeral-date-input-text"
+                        isDisabled={disabled}
+                      >
+                        {getDateLabel(datePart, locale)}
+                      </StyledDateLabel>
+                      <Textbox
+                        {...(index === 0 && { id: uniqueId })}
+                        disabled={disabled}
+                        readOnly={readOnly}
+                        required={required}
+                        error={!!internalError}
+                        warning={!!internalWarning}
+                        info={!!info}
+                        size={size}
+                        aria-labelledby={labelId}
+                        value={dateValue[datePart as keyof NumeralDateObject]}
+                        onChange={(e) =>
+                          handleChange(e, datePart as keyof NumeralDateObject)
+                        }
+                        onBlur={handleBlur}
+                        ref={(element) => handleRef(element, index, inputRef)}
+                        {...(isEnd &&
+                          !validationOnLabel && {
+                            error: internalError,
+                            warning: internalWarning,
+                            info,
+                          })}
+                        tooltipPosition={tooltipPosition}
+                      />
                     </StyledDateField>
                   </NumeralDateContext.Provider>
                 );
               })}
             </StyledNumeralDate>
+            {fieldHelp && <FieldHelp>{fieldHelp}</FieldHelp>}
           </Box>
-        </FormField>
-      </InputGroupBehaviour>
-    </TooltipProvider>
+        </StyledFieldset>
+      </TooltipProvider>
+    );
+  }
+
+  return (
+    <StyledFieldset
+      data-component={dataComponent}
+      data-element={dataElement}
+      data-role={dataRole}
+      id={uniqueId}
+      legend={label}
+      isRequired={required}
+      isOptional={isOptional}
+      isDisabled={disabled}
+      name={name}
+      aria-describedby={labelHelp && `${uniqueId}-label-help`}
+      {...filterStyledSystemMarginProps(rest)}
+    >
+      {labelHelp && (
+        <StyledHintText id={`${uniqueId}-label-help`}>
+          {labelHelp}
+        </StyledHintText>
+      )}
+
+      <Box position="relative" mt={1}>
+        {(internalError || internalWarning) && (
+          <>
+            <ValidationMessage
+              error={internalError}
+              warning={internalWarning}
+              validationId={`${uniqueId}-validation-message`}
+            />
+            <ErrorBorder warning={!!(!internalError && internalWarning)} />
+          </>
+        )}
+
+        <StyledNumeralDate onKeyDown={onKeyDown} data-component="numeral-date">
+          {dateFormat.map((datePart, index) => {
```

Comment:
suggestion (non-blocking): It looks like some of the composition here is the same as above, I wonder if it would be possible to store some of this in a variable and pass it to the relevant wrappers depending on the status of the validation flag.

```
const computedInputs = (...)

  if (!validationRedesignOptIn) {
   return (
     <TooltipProvider helpAriaLabel={helpAriaLabel}>
        <StyledFieldset
          ...
          >
          {computedInputs()}
        </StyledFieldset>
     </TooltipProvider>
   )
 }
 
 return (
    <StyledFieldset
       ...
     >
        {computedInputs()}
    </StyledFieldset>